lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,23 @@
+<div class="wrapper-msg is-incontext urgency-low warning">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%= gettext("Oops!") %></h3>
+            <div class="copy">
+                <p aria-live="polite"></p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="team-edit-fields">
+    <div class="team-required-fields">
+    </div>
+
+    <div class="team-optional-fields">
+    </div>
+</div>
+
+<div class="create-team form-actions">
+    <button class="action action-primary"><%- gettext("Create") %><span class="sr"><%- gettext("a new team") %></span></button>
```

Comment:
Please see our i18n coding guidelines: https://github.com/edx/edx-platform/wiki/i18n-Coding-Guidelines. This will confuse the translators as-is.